src/qml/qgismobileapp.qml

```diff
@@ -4012,4 +4044,52 @@ ApplicationWindow {
     id: screenLocker
     enabled: false
   }
+
+  Dialog {
+    id: pluginPermissionDialog
+    parent: mainWindow.contentItem
+
+    visible: false
+    modal: true
+    font: Theme.defaultFont
+
+    z: 10000 // 1000s are embedded feature forms, user a higher value to insure the dialog will always show above embedded feature forms
+    x: ( mainWindow.width - width ) / 2
+    y: ( mainWindow.height - height ) / 2
+
+    title: ''
+
+    Column {
+      Label {
+        width: parent.width
+        wrapMode: Text.WordWrap
+        text: qsTr( "Do you grant permission to activate this plugin?" )
```

Comment:
```suggestion
        text: qsTr( "Do you grant permission to activate the plugin `%1` in this project?" ).arg( pluginPermissionDialog.title )
```

Reply:
@m-kuhn , done.

Reply:
@m-kuhn , slightly modified though, because ATM a project plugin creates a "Project Plugin" title, which would result in a weird sentence as suggested (i.e. Do you grant permission to activate the plugin 'Project Plugin' in this project?)